$NetBSD: patch-configure,v 1.1 2011/05/02 13:39:40 wiz Exp $

Do not hardcode libdl, it doesn't exist on e.g. NetBSD. Use DL_LIBS
variable set by dlopen.builtlink.mk.

--- configure.orig	2011-05-02 13:26:47.000000000 +0000
+++ configure
@@ -6316,7 +6316,7 @@ else
   CFLAGS="$CFLAGS -fomit-frame-pointer"
 fi
 
-LIBS="$LIBS -lX11 -ldl -lXext"
+LIBS="$LIBS -lX11 ${DL_LIBS} -lXext"
 
 ZSNESFX='#ZSNESFX=yes'
 ZSNESC4='#ZSNESC4=yes'
